include/swift/AST/Decl.h

```diff
@@ -5216,6 +5216,11 @@ class ProtocolDecl final : public NominalTypeDecl {
   /// protocol.
   bool inheritsFrom(const ProtocolDecl *Super) const;
 
+  /// Determine whether this protocol has ~<target>` stated on
+  /// itself, one of its inherited types or `Self` requirements.
+  std::pair</*found=*/bool, /*where=*/SourceLoc>
+  hasInverseMarking(InvertibleProtocolKind ip) const;
```

Comment:
if CI fails, this would be nice to fix: name parameter `target` instead of `ip`.

Reply:
Done!